examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1514,9 +1527,9 @@ class GLTFWriter {
 
 			if ( ! validVertexAttributes.test( attributeName ) ) attributeName = '_' + attributeName;
 
-			if ( cache.attributes.has( this.getUID( attribute ) ) ) {
+			if ( cache.attributes.has( this.getUID( attribute, morphAttributes[ attributeName ] ) ) ) {
```

Comment:
Could we just say `this.getUID( attribute, false )` here? If there were a morph target version, we wouldn't want its ID here, correct? Also applies to the next two 'getUID' calls.